frontend/src/features/housingCompany/HousingCompanyCreatePage.tsx

```diff
@@ -142,6 +145,7 @@ const HousingCompanyCreatePage = (): JSX.Element => {
                     inputType="date"
                     label="Myyntihintaluettelon vahvistamispäivä"
                     fieldPath="sales_price_catalogue_confirmation_date"
+                    required
```

Comment:
this is not actually required. it's OK this is sent as `null` but not as blank string `""`

Reply:
Initial value is now set to `null`, and removed the `required` attribute from the field in question.

Reply:
works :+1: